crates/taplo/src/formatter/mod.rs

```diff
@@ -709,18 +715,21 @@ fn add_entries(
             row.push(e.value);
             if let Some(c) = e.comment {
                 row.push(c);
+                comment_count += 1;
             }
 
             row
         })
         .collect::<Vec<_>>();
 
+    let align_comments =
+        (comment_count != 1 || options.align_single_comments) && options.align_comments;
```

Comment:
nit: We should create a `fn should_align_comments(&self, comment_count: usize) -> bool` helper function in the `impl Options` line 168 of `src/formatter/mod.rs` because we use this logic both here and below.

Reply:
I considered it. Glad to know my intuition was aligned with yours :smile: